scenarios/kubernetes_e2e.py

```diff
@@ -578,6 +578,13 @@ def main(args):
             '--kubernetes-anywhere-kubeadm-version=%s' % version,
         ])
 
+        if args.kubeadm == "pull":
```

Comment:
I anticipate you will get a push not to do this in scenarios but rather to do this in kubetest (ie. https://github.com/kubernetes/test-infra/blob/master/kubetest/anywhere.go).

Idea: maybe we should support something like "kubeadm" in addition to "stable" and gs:// link for kubernetesAnywhereKubeletVersion where "kubeadm" means to match the version of kubeadm in anywhere.go.

Reply:
Yeah, I also want to get this out of scenarios eventually.
But for now, I want to get it working first.